app/util/files/FileControllerMixin.js

```diff
@@ -0,0 +1,151 @@
+﻿Ext.define('CpsiMapview.util.files.FileControllerMixin', {
```

Comment:
Does this have to be a mixin? Could it be the controller for CpsiMapview.util.files.FileUploadWindow, or is it used within other components?

Reply:
It implements event handlers also for the file grid (add new and delete files) so I think it's better as a mixin to be used in the controller of the window which includes the grid